modules/hcp-bm-add-np.adoc

```diff
@@ -19,7 +19,7 @@ $ hcp create nodepool agent \
   --cluster-name <hosted_cluster_name> \// <1>
   --name <nodepool_name> \// <2>
   --node-count <worker_node_count> \// <3>
-  --agentLabelSelector '{"matchLabels": {"size": "medium"}}' <4>
+  --agentLabelSelector size=medium <4>
```

Comment:
Should the description be 
<4> The `--agentLabelSelector` flag is optional. The node pool uses agents with the `size=medium` label ?

Do we need to update the callout4 description accordingly?

Reply:
Yes! Good catch. Thanks!